core/include/msg.h

```diff
@@ -148,7 +281,7 @@ int msg_send_int(msg_t *m, kernel_pid_t target_pid);
  */
 static inline int msg_sent_by_int(const msg_t *m)
 {
-    return (m->sender_pid == KERNEL_PID_ISR);
+    return m->sender_pid == KERNEL_PID_ISR;
```

Comment:
I think it's caused by `mod_paren_on_return = remove`, but setting this to ignore or even add would mean that e.g. `return (1);` would be correct, too.